### DISCUSSION TITLE
Future of vue-leaflet

### DISCUSSION THREAD

**Discussion by nosehair** [vue-leaflet/vue-leaflet#227] [General]
Are there any plans to actively maintain this project? Used vue2-leaflet in the past but have since migrated to vue3. Looking at the issues and commits it seems as though nothing is happening here.

**Comment by mikeu** [comment]
Hi @nosehair , yes, I do indeed have such plans. I've been absent from the project for quite a few months for various reasons, but am now ramping my involvement back up. I'm planning a new release to address the common problems around not working well with imports from `"leaflet"`, as in [this discussion](https://github.com/vue-leaflet/vue-leaflet/discussions/161), and will be making my way through the open PRs as part of that process as well.

**Reply by 1Luc1** [reply]
Any news on this topic?

**Comment by Morl99** [comment]
@mikeu @DonNicoJs greetings! We are currently in the migration from vue2 to vue3 and maintain a product with leaflet at its heart. I wonder if you would be open to ongoing contributions from us (I am speaking on behalf of my employer [DB Systel](https://github.com/dbsystel)). We are currently trying to figure out, if an investment in the leaflet ecosystem would be worthwile. Our alternative would be a redesign to another map framework like mapbox or openlayers. The main reason to switch would be the impression, that leaflet (or rather the vue leaflet) community is dead. It would be good, if we could start a conversation about this.

**Reply by 1Luc1** [reply]
Please keep me updated with your decision, since I face kind of the same problem. My thought so far: 

- **[vue-leaflet](https://github.com/vue-leaflet/vue-leaflet)**: not maintained
- **[vue3-openlayers](https://github.com/MelihAltintas/vue3-openlayers)**: same state as vue-leaflet
- **[vue-mapbox-gl](https://github.com/studiometa/vue-mapbox-gl)**: need to use Access tokens, which are (price) limited!?

**Reply by nmezzopera** [reply]
Hey Folks,  thanks for pinging me, as it is clear from my involvement that I do not have bandwith to maintain the project, I am actually not coding directly since a while.

**Reply by Morl99** [reply]
> Hey Folks,  thanks for pinging me, as it is clear from my involvement that I do not have bandwith to maintain the project, I am actually not coding directly since a while.

That makes sense. Would you be willing to add maintainers to the vue-leaflet org to keep the project alive?

**Reply by mikeu** [reply]
Yes thank you for the ping @Morl99 . I too have had less time than I'd like to work on this project this year, having spent the first half wrapping up my previous engagement and the second half ramping up at a new job that doesn't involve either Vue or Leaflet. That said, I am now in a position to commit to checking in here at least once a week from now on, and to being more actively involved in the development again as well. I have finally opened a branch for the simple [default behaviour rework](https://github.com/vue-leaflet/vue-leaflet/discussions/161) and will aim for v0.7 to include that soon. The bulk of the work will be around updated documentation end examples for the breaking change; the actual code change looks to be quite minor.

I'll be going through the open PRs we already have as well, and I would definitely welcome additional contributions, ongoing or otherwise. I also agree that it's a great idea to add another maintainer or two once there is more active development around here again, especially now that we officially know that DonNico has no plans to return.

**Reply by Morl99** [reply]
Okay, thank you for your responses. I will take these back to my team and we will discuss on how to continue. As usual I think the best idea is to promote active contributors to maintainers after a while, so the first course of action would be to start contributing, now that we now that contributions are still welcome (and will likely be accepted).

**Reply by 1Luc1** [reply]
any news on this topic?

**Comment by aentwist** [comment]
This project seems to require an owner, since the current maintainers are not (judging by their actions). Finding someone up to this task is the question.
